using-mobile-extensions/adobe-target/README.md

```diff
@@ -506,9 +382,14 @@ If you need to update SDK configuration, programmatically, use the following inf
 | target.environmentId | Environment ID you want to use, if this is left blank, the default production environment will be used. |
 | target.propertyToken | `at_property token` value, which is generated from the Target UI. If this value is left blank, no token is sent in the Target network calls. |
 | target.previewEnabled | Boolean parameter, which can be used to enable/disable Target Preview. If not specified, then Preview will be enabled by default. |
+| target.sessionTimeout | Time, in seconds, for which the Target session ID and Egde Host are persisted. If not specified, then default timeout value is 30 minutes. |
+
+{% hint style="warning" %}
+We recommend that you use Experience Platform Launch configuration to pass the property token instead of passing it in as a mbox parameter. If the property token is passed in Experience Platform Launch configuration and also as a mbox parameter, the token that was entered in the mbox parameter is discarded.
```

Comment:
We recommend that, instead of passing the property token in as a mbox parameter, you use an Experience Platform Launch configuration to pass the token. If the token is passed in an Experience Platform Launch configuration, and as a an mbox parameter, the token that was entered in the mbox parameter is discarded.